tests/contrib/aiohttp/test_middleware.py

```diff
@@ -333,8 +334,8 @@ def test_distributed_tracing_sub_span(self):
         # with the right trace_id and parent_id
         eq_(100, span.trace_id)
         eq_(42, span.parent_id)
-        eq_(0, span._sampling_priority)
+        eq_(0, span.get_metric(SAMPLING_PRIORITY_KEY))
         # check parenting is OK with custom sub-span created within server code
         eq_(100, sub_span.trace_id)
         eq_(span.span_id, sub_span.parent_id)
-        eq_(0, span._sampling_priority)
+        eq_(None, sub_span.get_metric(SAMPLING_PRIORITY_KEY))
```

Comment:
ok, that's `None` because we don't attach this value to all children. Good to me.